content/en/security/cloud_workload_security/_index.md

```diff
@@ -5,13 +5,44 @@ aliases:
+## Detects threats to your production workloads in real-time
 
-{{< img src="security/cws/workload_security_rules.png" alt="Cloud Workload Security detection rules in the Datadog app" width="100%">}}
+Monitor file and process activity at the kernel level to detect threats to your infrastructure, such as AWS EC2 instances, Docker containers, and Kubernetes clusters.
 
-Workload Security uses the Datadog Agent, so if you’re already using Datadog to monitor your environment (within the [supported versions and platforms][1]), there’s no need to provision additional resources or introduce new agents. If you don't already have the Datadog Agent set up, [start with setting up the Agent][2] on a [supported operating system][1]. As a part of the Datadog platform, you can combine real-time threat detection with metrics, logs, traces, and other telemetry to see the full context surrounding a potential attack on your workloads.
+CWS uses the Datadog Agent to monitor your environment. If you don't already have the Datadog Agent set up, [start with setting up the Agent][2] on a [supported operating system][1]. There are four types of monitoring that the Datadog Agent uses for Cloud Workload Security:
+
+1. **Process Execution Monitoring** to watch process executions for malicious activity on hosts or containers in real-time.
+2. **File Integrity Monitoring** to watch for changes to key files and directories on hosts or containers in real-time.
+3. **DNS Activity Monitoring** to watch network traffic for malicious activity on hosts and containers in real-time.
+4. **Kernel Activity Monitoring** to watch for kernel-layer attacks like process hijacking, container breakouts, and more in real-time.
+
+{{< img src="security/cws/cws_overview.png" alt="Cloud Workload Security overview page" width="100%">}}
+
+## Manage out-of-the-box and custom detection rules
+
+CWS comes with more than 50+ out-of-the-box detection rules that are maintained by a team of security experts. The rules surface the most important risks so that you can immediately take steps to remediate. Agent expression rules define the workload activities to be collected for analysis while backend detection rules analyze the activities and identify attacker techniques and other risky patterns of behavior.
+
+[Customize the rules][5] by defining how each rule monitors process, network, and file activity, [create custom rules][6], and [set up real-time notifications](#set-up-real-time-notifications) for new signals.
```

Comment:
```suggestion
[Customize the rules][5] by defining how each rule monitors process, network, and file activity.  [Create custom rules][6], and [set up real-time notifications](#set-up-real-time-notifications) for new signals.
```

Reply:
I'm going to keep this one as-is. There's additional content just before this sentence in the Remote Config branch. It will read a lot better once that branch is merged.